src/include/duckdb/storage/storage_info.hpp

```diff
@@ -67,11 +70,20 @@ struct MainHeader {
 	uint64_t version_number;
 	//! The set of flags used by the database
 	uint64_t flags[FLAG_COUNT];
-
 	static void CheckMagicBytes(FileHandle &handle);
 
+	string LibraryGitDesc() {
+		return string(library_git_desc, 0, 32);
```

Comment:
should this not use the MAX_VERSION_SIZE instead of `32` or are they not connected?